dataset.py

```diff
@@ -48,9 +49,13 @@ class DatasetGeographicPartitions:
 _TRAIN_VALIDATION_TEST_BOUNDS = [
     _TRAIN_FIXED_BOUNDS, _VALIDATION_FIXED_BOUNDS, _TEST_FIXED_BOUNDS]
 
+# For RANDOM data partition only, the fraction of the dataset allocated for train, validation
+# and test sets.
+TRAIN_VALIDATION_TEST_RATIOS = [0.8, 0.1, 0.1]
```

Comment:
nit:
a different way to do this that is a bit cleaner is to have two different subclasses for PartitionSettings - RandomPartitionSettings and BoundedPartitionSettings.

In the implementation of the method, you just upcast the single argument to the right setting.
this would make it clearer to the caller what the options are.

Reply:
I checked and we do need this. I've commited these changes.